### PR TITLE
[4.2.0] Add making non-admin user read-only config

### PR DESCRIPTION
**Pull request by DedunuKarunarathne**
## Purpose

$Subject

### CODE REVIEW

File: en/docs/install-and-setup/setup/user-stores/managing-users.md [hunk 1]
```diff
@@ -37,7 +37,17 @@ An existing admin user can log in to the Micro Integrator server from the CLI to
...
+
+#### Restrict non-admin users from updating the deployed artifacts
+To restrict the non-admin users from updating the deployed artifacts through **management API** and **CLI**, you can add the following configuration to the `<MI_HOME>/conf/deployment.toml` file. (Available from update level:4.2.0.103)
+
+```toml
+[user_access]
+make_non_admin_users_read_only = true
+```
+
+To restrict the non-admin users from updating the deployed artifacts with **MI dashboard**, you can add the same configuration to the `<MI_DASHBOARD_HOME>/conf/deployment.toml` file. (Available from update level:4.2.0.18)
```
Comment (chanikag) [inline en/docs/install-and-setup/setup/user-stores/managing-users.md:50]: shall we give the config here. Saying same configuration can be confusing.

Comment (DedunuKarunarathne) [reply en/docs/install-and-setup/setup/user-stores/managing-users.md]: Updated with d67de7fab40aa6e2691877a7c4b95d656b0fab8e

File: en/docs/install-and-setup/setup/user-stores/managing-users.md [hunk 2]
```diff
@@ -37,7 +37,22 @@ An existing admin user can log in to the Micro Integrator server from the CLI to
 
 ### Non-admin users
 
-Users that do not have admin privileges can access the management API, the CLI, and the dashboard to view and monitor integration artifacts and logs.
+Users that do not have admin privileges can access the management API, the CLI, and the dashboard to update the deployed artifacts, view and monitor integration artifacts and logs.
```
Comment (DinithiDiaz) [inline en/docs/install-and-setup/setup/user-stores/managing-users.md:40]:
```suggestion
Users without admin privileges can access the management API, the CLI, and the dashboard to update deployed artifacts, and to view and monitor integration artifacts and logs.
```

File: en/docs/observe-and-manage/working-with-monitoring-dashboard.md [hunk 3]
```diff
@@ -35,6 +35,14 @@ You can use the dashboard to perform the following administration tasks related
 
     You can enable/disable tracing for the following artifacts: <i>Proxy Services</i>, <i>Endpoints</i>, <i>APIs</i> <i>Sequences</i> and <i>Inbound Endpoints</i>.
 
+    !!! INFO
+        To restrict the non-admin users in MI dashboard from updating the deployed artifacts, you can add the following configuration to the `<MI_DASHBOARD_HOME>/conf/deployment.toml` file.
```
Comment (DinithiDiaz) [inline en/docs/observe-and-manage/working-with-monitoring-dashboard.md:39]:
```suggestion
        To restrict the non-admin users in the MI dashboard from updating the deployed artifacts, you can add the following configuration to the `<MI_DASHBOARD_HOME>/conf/deployment.toml` file.
```

**Review by DinithiDiaz — APPROVED** [review]

**Pull request merged**